### PR TITLE
feat(trading): navigation

### LINKED ISSUE #4296: Trading navigation changes
## The Chore

Redesign of navigation

## Tasks

- [x] Add/Adjust any specs
- [x] Dev
- [x] Test

## Additional details / background info

[Figma](https://www.figma.com/file/ywcCZvXdvG28bg0AmCY9mF/Console-v3?type=design&node-id=3166-5839&mode=design&t=vtxKpa25NYpqqGrL-0)

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

Closes #4296

# Description ℹ️

Adds bespoke nav component for trading app. Largely based on the current ui-toolkit version but with some simplification

# Demo 📺

Screenshot 2023-07-27 at 10 02 36

Screenshot 2023-07-27 at 09 59 08

Screenshot 2023-07-27 at 10 00 17

Screenshot 2023-07-27 at 10 00 49

# Technical 👨‍

- Expands trading navbar component to be bespoke for the trading app. 
- Adds adjusted TradingDropdown component so that any element can be used as the trigger, and so it has the right colors
- Rotates sidebar to the bottom of the screen on mobile
- Adds market selector to main nav child so it displays in the header on mobile
- Deletes navbar.cy.ts test and moves tests to console-test
- Adjusts colors of connected wallet dropdown

### CODE REVIEW

File: apps/trading/components/navbar/nav-header.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const { data } = useMarket(marketId);
+  const [open, setOpen] = useState(false);
+
+  if (!marketId) return null;
+
+  return (
+    <FullScreenPopover
+      open={open}
+      onOpenChange={(x) => {
+        console.log(x);
```
Comment (bglownia) [inline apps/trading/components/navbar/nav-header.tsx:23]: We should add no-console eslint rule long time ago

File: apps/trading/components/vega-wallet/vega-wallet-menu.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}) => {
+  const [copied, setCopied] = useCopyTimeout();
+
+  return (
+    <div
+      className="flex flex-col w-full ml-4 mr-2 mb-4"
+      data-testid={`key-${pk.publicKey}-mobile`}
+    >
+      <span className="flex gap-2 items-center mr-2">
+        <button onClick={() => onSelectItem(pk.publicKey)}>
```
Comment (bglownia) [inline apps/trading/components/vega-wallet/vega-wallet-menu.tsx:85]: missing type attribute

File: apps/trading/components/vega-wallet/vega-wallet-menu.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          <span className="uppercase">{pk.name}</span>
+        </button>
+        {isActive && <VegaIcon name={VegaIconNames.TICK} />}
+      </span>
+      <span className="flex gap-2 items-center">
+        {truncateByChars(pk.publicKey)}{' '}
+        <CopyToClipboard text={pk.publicKey} onCopy={() => setCopied(true)}>
+          <button
```
Comment (bglownia) [inline apps/trading/components/vega-wallet/vega-wallet-menu.tsx:93]: missing type attribute

File: libs/ui-toolkit/src/components/trading-dropdown/actions-dropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  TradingDropdownTrigger,
+} from './trading-dropdown';
+
+export const ActionsDropdownTrigger = () => {
+  return (
+    <TradingDropdownTrigger
+      className='hover:bg-vega-light-200 dark:hover:bg-vega-dark-200 [&[aria-expanded="true"]]:bg-vega-light-200 dark:[&[aria-expanded="true"]]:bg-vega-dark-200 p-0.5 rounded-full'
+      data-testid="dropdown-menu"
+    >
+      <button>
```
Comment (bglownia) [inline libs/ui-toolkit/src/components/trading-dropdown/actions-dropdown.tsx:14]: missing type attribute

File: apps/trading/components/navbar/nav-header.tsx (hunk 1 above)
Comment (mattrussell36) [reply apps/trading/components/navbar/nav-header.tsx]: Added a default console log rule (which still allows warn/error)

**Pull request merged**